args.go

```diff
@@ -16,6 +16,7 @@ package cobra
 
 import (
 	"fmt"
+	"github.com/leonelquinteros/gotext"
```

Comment:
We could import this as `i18n`, so that instead of calling `gotext.Get()` we would call `i18n.Get()`.
This may be clearer when browsing the code?